HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/util/CertificateStringMapBuilder.java

```diff
@@ -56,9 +64,27 @@ private CertificateStringMapBuilder() {
                 data.put("isSelfSigned", "false");
             }
 
+            data.put("authKeyId", certificate.getAuthKeyId());
+            data.put("authInfoAccess", certificate.getAuthInfoAccess());
+            data.put("signatureAlgorithm", certificate.getSignatureAlgorithm());
             if (certificate.getEncodedPublicKey() != null) {
                 data.put("encodedPublicKey",
                         Arrays.toString(certificate.getEncodedPublicKey()));
+                data.put("publicKeyAlgorithm", certificate.getPublicKeyAlgorithm());
+            }
+
+            if (certificate.getPublicKeyModulusHexValue() != null) {
+                data.put("publicKeyValue", certificate.getPublicKeyModulusHexValue());
+                data.put("publicKeySize", null);
```

Comment:
At the moment I want to leave it in as a place holder.  This is information requested to be displayed by @iadgovuser26 but the current library is pulling an additional 4 bytes that are not a part of the key.  If this can be resolved in time, it will be displayed.

Reply:
BouncyCastle.

Reply:
Alrighty, I'll defer to you on this one.